mat64/matrix.go

```diff
@@ -382,15 +382,7 @@ func MaybeFloat(fn FloatPanicker) (f float64, err error) {
 	return fn(), nil
 }
 
-// Must can be used to wrap a function returning an error.
-// If the returned error is not nil, Must will panic.
-func Must(err error) {
-	if err != nil {
-		panic(err)
-	}
-}
-
-// Type Error represents matrix package errors. These errors can be recovered by Maybe wrappers.
+// Type Error represents matrix handling errors. These errors can be recovered by Maybe wrappers.
```

Comment:
Oops, didn't realized this was already merged. You can take care of this when you like.